openfecwebapp/templates/partials/candidate/about-candidate.html

```diff
@@ -107,7 +107,7 @@ <h2 id="section-2-heading">About this candidate</h2>
       {% for committee in committee_groups['P'] | reverse %}
         <div class="callout callout--primary{% if loop.last %} u-no-margin{% endif %}">
           <h4 class="callout__title">
-            <a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle) }}">{{ committee.name }}</a>
+            <a href="{{ url_for('committee_page', c_id=committee.committee_id) }}">{{ committee.name }}</a>
```

Comment:
Ack, sorry, I don't think I explained that well. We only want to exclude the cycle parameter if we're looking at the most recent time period. That way if you're looking at historical data, it still takes you to the historical committee page. Or does that bring up all kinds of weird logic that would just make it a mess?